docs/source/tutorial.rst

```diff
@@ -0,0 +1,659 @@
+Tutorial
+========
+
+This tutorial is intended for beginners with bats and possibly bash.
+Make sure to also read the list of gotchas and the faq.
+
+For this tutorial we are assuming you already have a project in a git repository and want to add tests.
+Ultimately they should run in the CI environment but will also be started locally during development.
```

Comment:
> Ultimately they should run in the CI environment but will also be started locally during development.

This is a subjective thing - I believe that particular sentence is a bit out of place since the writing of the tests will be the same process whether or not it runs locally or in CI. Maybe we can add a separate TODO section for "Example of bats in CI"?